install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -56,40 +86,102 @@ mkdir $HOME/.local/share/odemis/log 2>/dev/null
 parse_logfile=$HOME/.local/share/odemis/log/odemis-select-mic-start-parse.log
 > $parse_logfile
 
-for fn in "${args[@]}"; do
-    # add a new header to the logfile
-    printf "$(date) Parsing file: $fn\n" >> $parse_logfile
-    # extract the microscope name by using this small Python script
+process_file() {
+    local fn="$1"
+
+    # Add a new header to the logfile
+    printf "$(date) Parsing file: $fn\n" >> "$parse_logfile"
+
+    # Extract the microscope name using a small Python script
     mic_name="$(python3 -c "import sys; import yaml; from odemis.odemisd import modelgen; \
         f = open('$fn'); \
         d = yaml.load(f, modelgen.SafeLoader); \
-        print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))" 2>>$parse_logfile)"
+        print(next(n for n, d in d.items() if d.get('class') == 'Microscope'))" 2>>"$parse_logfile")"
+
     mic_list+=("$fn")
+
     if test "$mic_name" == ""; then
         # save the file name in the error list
         error_filenames+=("$(basename ${fn})")
         mic_name=">>ERROR parsing $(basename ${fn})"
         # add a newline to the logfile
         echo "" >> $parse_logfile
     fi
+
     mic_list+=("$mic_name")
-done
 
-# prompt the user with a list of microscope configurations to choose from
-selection="$(zenity --list --title="Odemis configuration starter" \
-    --list --text "Please select a configuration to start:" \
-    --column "" --column "" "${mic_list[@]}" --hide-header \
-    --width=300 --height=400 \
-    --hide-column 1 --print-column=1 2>/dev/null)"
+    # Format the setup name
+    setup_name="${fn#*-}"
+    setup_name="${setup_name%.odm.yaml}"
+    mic_list+=("$setup_name")
+}
+
+# Check if the input is a directory or a wildcard
+if [ -d "${args[0]}" ]; then
+    # Input is a directory, list YAML files and extract unique prefixes
+    yaml_files=("${args[0]}"/*.yaml)
```

Comment:
Let's not support passing a directory, to keep things simple. The user has to pass filenames, always.